### DISCUSSION TITLE
Long standing open issues a concern for MLOps with ADO investment?

### DISCUSSION THREAD

**Discussion by corticalstack** [Azure/mlops-v2#103] [General]
HI,

Given there are long-standing open issues from middle of last year on this repo, it's a concern around Microsoft roadmap for MLOps with ADO and continuous improvement of this solution accelerator and supporting educational materials.

Would welcome feedback, thanks

**Comment by setuc** [comment]
This repo is under active development, and we are fixing major bugs as we come across. Some places where we dont have much details on the issue or find it hard to reproduce it are left open. The issue log certainly needs a cleanup. 

Is there a particular issue that's preventing you from ADO adoption?
